Description: Change error messages for vboxdrv failure, according to virtualbox-ose-dkms.
Author: Daniel Hahler <ubuntu@thequod.de>

diff -urNad virtualbox-ose-4.0.2-dfsg/src/VBox/VMM/VMMR3/VM.cpp virtualbox-ose-4.0.2-dfsg/src/VBox/VMM/VMMR3/VM.cpp
--- virtualbox-ose-4.0.2-dfsg/src/VBox/VMM/VMMR3/VM.cpp	2011-01-20 11:55:54.000000000 +0100
+++ virtualbox-ose-4.0.2-dfsg/src/VBox/VMM/VMMR3/VM.cpp	2011-01-20 11:56:18.215472116 +0100
@@ -289,10 +289,8 @@
 
 #ifdef RT_OS_LINUX
                 case VERR_SUPDRV_COMPONENT_NOT_FOUND:
-                    pszError = N_("One of the kernel modules was not successfully loaded. Make sure "
-                                  "that no kernel modules from an older version of VirtualBox exist. "
-                                  "Then try to recompile and reload the kernel modules by executing "
-                                  "'/etc/init.d/vboxdrv setup' as root");
+		    pszError = N_("Please install the virtualbox-dkms package "
+				  "and execute 'modprobe vboxdrv' as root");
                     break;
 #endif
 
@@ -334,10 +332,8 @@
             {
                 case VERR_VM_DRIVER_LOAD_ERROR:
 #ifdef RT_OS_LINUX
-                    pszError = N_("VirtualBox kernel driver not loaded. The vboxdrv kernel module "
-                                  "was either not loaded or /dev/vboxdrv is not set up properly. "
-                                  "Re-setup the kernel module by executing "
-                                  "'/etc/init.d/vboxdrv setup' as root");
+                    pszError = N_("Please install the virtualbox-dkms package "
+		    		  "and execute 'modprobe vboxdrv' as root");
 #else
                     pszError = N_("VirtualBox kernel driver not loaded");
 #endif
@@ -376,10 +372,8 @@
                 case VERR_INVALID_HANDLE: /** @todo track down and fix this error. */
                 case VERR_VM_DRIVER_NOT_INSTALLED:
 #ifdef RT_OS_LINUX
-                    pszError = N_("VirtualBox kernel driver not installed. The vboxdrv kernel module "
-                                  "was either not loaded or /dev/vboxdrv was not created for some "
-                                  "reason. Re-setup the kernel module by executing "
-                                  "'/etc/init.d/vboxdrv setup' as root");
+                    pszError = N_("Please install the virtualbox-dkms package "
+		    		  "and execute 'modprobe vboxdrv' as root");
 #else
                     pszError = N_("VirtualBox kernel driver not installed");
 #endif
